src/Jsend/ServiceStatusResponse.php

```diff
@@ -100,6 +100,72 @@ public function getResponseBody(): array
         ];
     }
 
+    public static function checkServices(
+        array $servicesClients,
```

Comment:
instead of array[name => ClientInterface] consider passing here list with our Assertis\Http\Clients. I would first extend https://github.com/assertis/http-lib/blob/master/src/Client/Client.php by adding `?string $name`